packages/nimble-components/src/list-option/index.ts

```diff
@@ -66,6 +66,13 @@ export class ListOption extends FoundationListboxOption {
         super.connectedCallback();
         const owner = this.getListOptionOwner();
         owner?.registerOption(this);
+        const observer = new MutationObserver(_ => {
+            this.$emit('display-changed');
+        });
+        observer.observe(this, {
```

Comment:
This runs contrary to the above (because I want to avoid creating a huge number of unnecessary mutation observers) but just fyi FAST has a first class pattern for registering mutation observers via the `children` directive.